File: Assets/InsightARWorld/InsightARExporter/Utility/BuiltinLuaScripts/Landmark/LandmarkIdleState.lua



local LandmarkIdleState = {}


function LandmarkIdleState:New(game_object)
    if self ~= LandmarkIdleState then
        return nil, "LandmarkIdleState:New is nil"
    end
    local newinstance = setmetatable({}, {__metatable = {}, __index = LandmarkIdleState})
    newinstance.transform = game_object.transform
    transform = game_object.transform
    newinstance.gameObject = game_object
    gameObject = game_object
    return newinstance
end

function LandmarkIdleState:Awake()
	
	
end

function LandmarkIdleState:Start()
	
end

function LandmarkIdleState:Enter()
	
end

function LandmarkIdleState:Execute()

end

function LandmarkIdleState:Exit()
	
end

function LandmarkIdleState:Init()

end



return LandmarkIdleState

File: U3DExporter/2022_10_29_10_04_55_Android/Assets/InsightARWorld/InsightARExporter/Utility/BuiltinJSScripts/Landmark/LandmarkIdleState.js
function LandmarkIdleState(gameObject)
{
    this.transform = gameObject.transform;
    this.game_object = gameObject;
}
//Write prototype function here
LandmarkIdleState.prototype = Object.assign(Object.create(Object.prototype), {
    // Start is called before the first frame update
    Start: function(){
        
    },
    Awake: function(){

    },
    // Update is called once per frame
    Update: function()
    {
        
    },
    Enter: function()
	{
    },
    Execute: function()
    {

    },
    Exit: function()
	{

    },
    Init: function()
    {

    },
    Close: function () {

    }
});

//Return the script module
LandmarkIdleState